src/main/webapp/statistics.html

```diff
@@ -7,22 +7,42 @@
     <h1>Statistics</h1>
     <script type="text/javascript"
     src="https://www.gstatic.com/charts/loader.js"></script>
+    <h2>General</h2>
+    <ul>
+      <li>19% of all pregnancies end in abortion</li>
+      <li>1 in 3 American women will have an abortion by age 45</li>
+      <li>6 in 10 women having abortions already have a child</li>
+    </ul>
     <script type="text/javascript">
 
     //load Charts and the corechart package.
     google.charts.load('current', {'packages': ['corechart']});
 
+    //Draw the pie chart  for racial distribution of abortions
```

Comment:
For formatting comment, add a space after //. // Draw the pie chart  for racial distribution of abortion
Make the change here and in other comments

Reply:
implemented check